packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/pre_test.sh

```diff
@@ -3,9 +3,9 @@
 . ./upgrade-test-scripts/env_setup.sh
 
 # ensure there's only uist
+test_val "$(agd q bank balances agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346 -o json | jq -r '.balances | length')" "1"
```

Comment:
_note_:
The agoric1megzy… address is the account of the provisioning pool.

Reply:
Yes, I copied this over from other tests to ensure it's the same across upgrades. There's an identical test in `agoric-upgrade-8`, after it's funded.